samples/ClassroomLabs/Modules/Library/Az.LabServices.psm1

```diff
@@ -829,13 +829,13 @@ function New-AzLab {
                 $environmentSettingUri = $labUri + "/environmentsettings/default"
                 $sharedPassword = if ($SharedPasswordEnabled) { "Enabled" } else { "Disabled" }
                 $imageType = if ($image.id -match '/galleryimages/') { 'galleryImageResourceId' } else { 'sharedImageResourceId' }
-                if ($LinuxRdpEnabled) {$linuxRdpState = 'Enabled'} else { $linuxRdpState = 'Disabled' }
-                if ($InstallGpuDriverEnabled) {$gpuDriverState = 'Enabled'} else { $gpuDriverState = 'Disabled' }
+                if ($LinuxRdp) {$linuxRdpState = 'Enabled'} else { $linuxRdpState = 'Disabled' }
+                if ($GpuDriverEnabled) {$gpuDriverState = 'Enabled'} else { $gpuDriverState = 'Disabled' }
                 if ($idleGracePeriod -eq 0) {$idleShutdownMode = "None"} else {$idleShutdownMode = "OnDisconnect"}
                 if ($idleOsGracePeriod -eq 0) {$enableDisconnectOnIdle = "Disabled"} else {$enableDisconnectOnIdle = "Enabled"}
                 if ($idleNoConnectGracePeriod -eq 0) {$enableNoConnectShutdown = "Disabled"} else {$enableNoConnectShutdown = "Enabled"}
 
-                if ($LinuxRdpEnabled) {
+                if ($LinuxRdp) {

```

Comment:
This code looks wrong - but it's probably fine if we don't address it right now (maybe log a task for it).  It looks like we are deciding to set the Idle settings IF the LinuxRDP property is passed.  I don't think that's right because some Linux operating systems do support idle settings.  We should either be asking the user if it's a LinuxOS (and making both the RDP & idle settings decision based on that), or have 2 properties...  We can fix this later though, definitely don't need to hold up the PR for it.

Reply:
Ah yes - idle setting support for some Linux OS's wasn't added until fairly recently, so this code never got updated to reflect that. I'll log a task for this as well as part of the vNext rework.